actors/test/actor-tests/src/test/java/cloud/orbit/actors/test/DeactivationTest.java

```diff
@@ -63,6 +63,7 @@ public void simpleCleanupTest() throws ExecutionException, InterruptedException
         final UUID id = actor1.getUniqueActivationId().join();
         clock.incrementTime(20, TimeUnit.MINUTES);
         stage.cleanup().join();
+        waitForDeactivations();
```

Comment:
The new LocalObjectCleaner cf just represents the submission of the deactivation request. It does not wait for any of them to be complete. 
The old cleanup would await the actual deactivations.